src/Packetery/Module/PacketaWidgetIntegration.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * PacketaWidgetIntegration.
+ *
+ * @package Packetery
+ */
+
+namespace Packetery\Module;
+
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationInterface;
+
+if ( file_exists( WP_PLUGIN_DIR . '/woocommerce/src/Blocks/Integrations/IntegrationInterface.php' ) ) {
+	require_once WP_PLUGIN_DIR . '/woocommerce/src/Blocks/Integrations/IntegrationInterface.php';
+}
+
+if ( file_exists( WP_PLUGIN_DIR . '/woocommerce/packages/woocommerce-blocks/src/Integrations/IntegrationInterface.php' ) ) {
+	require_once WP_PLUGIN_DIR . '/woocommerce/packages/woocommerce-blocks/src/Integrations/IntegrationInterface.php';
+}
+
+/**
+ * PacketaWidgetIntegration.
+ *
+ * @package Packetery
+ */
+class PacketaWidgetIntegration implements IntegrationInterface {
+	const VERSION          = '0.1.0';
```

Comment:
Toto je označení verze toho bloku ?  Nebylo by lepší to sjednotit s verzí pluginu ?